libbeat/beat/template.go

```diff
@@ -0,0 +1,67 @@
+package beat
+
+/*
+
+Load the index template. This replaces the manual running of
+curl -XPUT 'http://localhost:9200/_template/filebeat?pretty' -d@/etc/filebeat/filebeat.template.json
+
+
+TODO:
+ - Cleanup TODO's
+ - Should the template also be automatically loaded by logstash-input-beats?
```

Comment:
I think for logstash is done via the Elasticsearch output plugin. So we'd have to provide it in our sample LS config.